Expensify/App
# feat: Dynamic emoji size

### Details


### Fixed Issues

Fixes #2662

### Tests


### QA Steps


- Check for emoji sizes in small devices.
- Check for emoji overflow/clipping
### Tested On


### Screenshots


#### Web


<img width="1440" alt="Screenshot 2021-07-03 at 12 11 19 AM" src="https://user-images.githubusercontent.com/32242596/124317561-ffe87800-db94-11eb-9100-1fdd98621b1f.png">

#### Mobile Web

<img width="376" alt="Screenshot 2021-07-03 at 12 11 57 AM" src="https://user-images.githubusercontent.com/32242596/124317550-fbbc5a80-db94-11eb-8eea-fe8149f2f13b.png">

#### Desktop


<img width="1214" alt="Screenshot 2021-07-03 at 12 23 49 AM" src="https://user-images.githubusercontent.com/32242596/124317541-f8c16a00-db94-11eb-98f2-1cb89a9980d9.png">


#### iOS


![Simulator Screen Shot - iPhone 12 - 2021-07-02 at 22 09 58](https://user-images.githubusercontent.com/32242596/124310026-3ddf9f00-db89-11eb-82f2-0b38754442cc.png)


#### Android



Smallest device possible (approx 320)
![Screenshot from 2021-06-28 22-40-58](https://user-images.githubusercontent.com/32242596/123677075-4fac0400-d862-11eb-980d-02ddf42ab900.png)

Medium Size WVGA
![Screenshot from 2021-06-28 22-41-21](https://user-images.githubusercontent.com/32242596/123677126-62263d80-d862-11eb-8107-3fd9e95c4a24.png)

High DPI
![Screenshot from 2021-06-28 22-43-08](https://user-images.githubusercontent.com/32242596/123677292-900b8200-d862-11eb-89b2-8d580081f1f1.png)
